app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "neetoui";
+import PropTypes from "prop-types";
+
+import {
+  formatDateToDayAndTime,
+  formatDateToTimeSinceFromNow,
+} from "utils/index";
```

Comment:
No need to mentioned `/index` when importing from `index` file. Webpack will auto resolve it.